src/components/Nav/Nav.scss

```
@@ -0,0 +1,160 @@
+@font-face {
+  font-family: 'GangwonEduPowerExtraBoldA';
+  src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_2201-2@1.0/GangwonEduPowerExtraBoldA.woff')
+    format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@mixin display {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.navBox {
+  @include display;
+  position: relative;
+  align-items: center;
+  width: 100%;
+  height: 100px;
+  background-color: black;
+
+  .navLogo {
+    width: 80px;
+    height: 30px;
+
+    font: 28px 'GangwonEduPowerExtraBoldA';
+
+    .logo {
+      color: #fff;
+      text-decoration: none;
+    }
+  }
+  .navTitle {
+    @include display;
+    width: 700px;
+    color: #fff;
+    font-size: 16px;
+    cursor: pointer;
+
+    a {
+      color: #fff;
+      text-decoration: none;
+    }
+    .navCateTitle {
+      position: absolute;
+      top: 100px;
+
+      height: 300px;
+
+      .navInner {
+        position: relative;
+        display: flex;
+
+        &::before {
+          content: '';
+          position: absolute;
+          top: 0;
+          right: 50%;
+          width: 500%;
+          height: 300px;
+          background-color: #333;
+          margin-right: -100%;
+        }
+        &:after {
+          content: '';
+          position: absolute;
+          top: 0;
+          left: 0;
+          margin-top: -10px;
+          width: 0;
+          height: 0;
+          border-bottom: 10px solid #333;
+          border-left: 13px solid transparent;
+          border-right: 13px solid transparent;
+        }
+
+        .cateTitle {
+          position: relative;
+          top: 0;
+          left: -10px;
+          width: 300px;
+          height: 300px;
+          padding-left: 10px;
+          background-color: #333;
+
+          .title {
+            margin: 30px 0;
+            font-size: 18px;
+            font-weight: bold;
+          }
+          .cateList {
+            color: #717171;
+            font-size: 15px;
+            line-height: 2;
+          }
+          .cateList:hover {
+            color: #fff;
+            transition: all 0.8s;
+          }
```

& 가상 선택자 이용해서 다음처럼 할 수 있습니다!
```suggestion
          .cateList {
            color: #717171;
            font-size: 15px;
            line-height: 2;
            
            &:hover {
              color: #fff;
              transition: all 0.8s;
            }
          }
```